libs/features/pages/home/src/lib/main/home/home-page.component.html

```diff
@@ -3,4 +3,22 @@
     <elewa-website-projects-section></elewa-website-projects-section>
     <elewa-website-news-section></elewa-website-news-section>
   </div>
-</elewa-website-main-page>
\ No newline at end of file
+</elewa-website-main-page>
+<div class="home-page">
+  <elewa-website-header></elewa-website-header>
+  <elewa-website-home-hero-section></elewa-website-home-hero-section>
+  <elewa-website-news-section></elewa-website-news-section>
+  <elewa-website-footer>
+    <div footer class="footer-content">
+      <div class="footer-text">
+        <h4>
+          <span class="first-line">We’d love to build something</span>
+          <span class="second-line">amazing together!</span>
+        </h4>
+        <div class="mission">
+          <elewa-website-slider-button [buttonData]="buttonData"></elewa-website-slider-button>
+        </div>
+      </div>
+    </div>
+  </elewa-website-footer>
+</div>
```

Comment:
This was wrapped by the main page component. you should leave that way. Just add your component where it should be.